StripeCore/StripeCore/Source/Helpers/Async.swift

```diff
@@ -114,9 +114,10 @@ import Foundation
     }
 
     public func transformed<T>(
+        on queue: DispatchQueue = .main,
         with closure: @escaping (Value) throws -> T
     ) -> Future<T> {
-         chained { value in
+        chained(on: queue) { value in
```

Comment:
Drive-by cleanup of this [recently merged method](https://github.com/stripe/stripe-ios/pull/4051)!